nextcloud/content.md

```diff
@@ -499,99 +545,65 @@ The `--pull` option tells docker to look for new versions of the base image. The
 You're already using Nextcloud and want to switch to docker? Great! Here are some things to look out for:
 
 1.	Define your whole Nextcloud infrastructure in a `docker-compose` file and run it with `docker-compose up -d` to get the base installation, volumes and database. Work from there.
-
 2.	Restore your database from a mysqldump (nextcloud\_db\_1 is the name of your db container)
-
-	-	To import from a MySQL dump use the following commands
-
-	```console
-	docker cp ./database.dmp nextcloud_db_1:/dmp
-	docker-compose exec db sh -c "mysql --user USER --password PASSWORD nextcloud < /dmp"
-	docker-compose exec db rm /dmp
-	```
-
-	-	To import from a PostgreSQL dump use to following commands
-
-	```console
-	docker cp ./database.dmp nextcloud_db_1:/dmp
-	docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
-	docker-compose exec db rm /dmp
-	```
-
+	-	To import from a MySQL dump use the following commands`console
+		docker cp ./database.dmp nextcloud_db_1:/dmp
+		docker-compose exec db sh -c "mysql --user USER --password PASSWORD nextcloud < /dmp"
+		docker-compose exec db rm /dmp
+		`
+	-	To import from a PostgreSQL dump use to following commands`console
+		docker cp ./database.dmp nextcloud_db_1:/dmp
+		docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
+		docker-compose exec db rm /dmp
+		`
```

Comment:
This definitely isn't right :sweat_smile:  I think maybe something like this?

```suggestion
	-	To import from a MySQL dump use the following commands

		```console
		docker cp ./database.dmp nextcloud_db_1:/dmp
		docker-compose exec db sh -c "mysql --user USER --password PASSWORD nextcloud < /dmp"
		docker-compose exec db rm /dmp
		```

	-	To import from a PostgreSQL dump use to following commands

		```console
		docker cp ./database.dmp nextcloud_db_1:/dmp
		docker-compose exec db sh -c "psql -U USER --set ON_ERROR_STOP=on nextcloud < /dmp"
		docker-compose exec db rm /dmp
		```

```